modules.d/90multipath/module-setup.sh

```diff
@@ -21,14 +21,23 @@ majmin_to_mpath_dev() {
 
 # called by dracut
 check() {
+    local _any_mpath_dev
+
+    for_each_host_dev_and_slaves is_mpath
+    _any_mpath_dev=$?
+
     [[ $hostonly ]] || [[ $mount_needs ]] && {
-        for_each_host_dev_and_slaves is_mpath || return 255
+        [[ $_any_mpath_dev == 0 ]] || return 255
     }
 
     # if there's no multipath binary, no go.
     require_binaries multipath || return 1
     require_binaries kpartx || return 1
 
+    if [[ $_any_mpath_dev != 0 ]] && [[ ! -f /etc/multipath.conf ]]; then
```

Comment:
Just ftr it's better to use `-ne` for integers instead of `!=` in bash (the latter is inteded for strings).

Reply:
Probably `-gt` since we do not want to act on a negative count of mpath devices. ;)